sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -143,6 +143,22 @@ extern "C" {
 #endif
 #endif
 
+/**
+ * @brief   Include a Route Information Option for subnets
+ *          on other interfaces in normal Router Advertisements
+ *          generated by @ref gnrc_ipv6_nib_change_rtr_adv_iface
+ *
+ *          This is only needed if your node is an upstream router,
+ *          but not the default router, but you want to propagate
+ *          the information that the custom subnets it knows about
+ *          should be routed through it instead of the default route.
+ *
+ *          Requires the `gnrc_ipv6_nib_rio` module.
+ */
+#ifndef CONFIG_GNRC_IPV6_NIB_ADD_RIO_IN_RA
```

Comment:
And maybe set this to 1 if `IS_USED(MODULE_GNRC_IPV6_NIB_RIO)`.
And use `IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_ADD_RIO_IN_RA)`.

Reply:
But we don't always want this enabled when there is RIO support enabled - that could just be to receive RIOs.

The option is really only needed when we are an upstream router but not a default router - that's rather too specific IMHO to enable it by default.

Reply:
just asking ...

Reply:
I don't see how it adds value - the definition in the header already ensures that the define is always defined, so wrapping it in an `IS_ACTIVE()` is just unnecessary clutter IMHO

Reply:
Oh ok I thought it would be good RIOT practice. I can then just omit it in the future.